src/_utils/rideAxis/RideAxis.story.mdx

```diff
@@ -2,7 +2,7 @@ import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks'
 
 import { RideAxis } from './index'
 
-<Meta title="Rides/RideAxis" />
+<Meta title="Widgets/RideAxis" />
```

Comment:
RideAxis could be in a category "Data display" with itinerary, profile, trip card, avatar, etc.

Reply:
IMHO Data Display is a way too large category almost like Widgets and Miscelanious since components deal with an API attached to the product itself.

We have already a JIRA ticket to improve discoverability. And It will be tackle on that ticket. 

I've changed here because they where "orfans" on our actual taxonomy